scylladb/sphinx-scylladb-theme
# chore: replace localstorage with cache

Closes #1230

Closes #1231

This update switches from using local storage to cookies for storing user preferences across different domains. Additionally, it ensures that system preferences for dark mode are respected.

By definition, cookies are shared between domains, whereas local storage is not.

## How to test

Testing locally can be challenging since we don't have two projects using the updated theme. However, you can verify that preferences for the dark theme and the expanded bar are retained across domains using the same theme. You can also manually define the dark theme preference in Chrome using [this guide](https://support.google.com/chrome/answer/9275525?hl=en&co=GENIE.Platform%3DDesktop).

To perform this test, ensure you're in incognito mode with a clean cache.
